include/hip/nvcc_detail/hip_runtime_api.h

```diff
@@ -655,11 +655,20 @@ inline static hipError_t hipMemset(void* devPtr, int value, size_t count) {
     return hipCUDAErrorTohipError(cudaMemset(devPtr, value, count));
 }
 
+inline static hipError_t hipMemsetD32(void* devPtr, int value, size_t count) {
```

Comment:
Breaks build on NVCC path. This should be hipDeviceptr_t devPtr.

Reply:
done